### PR TITLE
WIP: New Colors 1: Updated nav on desktop

### PR DESCRIPTION
**Pull request by chrisnojima**
This is going to be a series of PRs, i don't want to merge it until whole sets of it are done (desktop and mobile)

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 1b0ee2e07ac6...0828ac733081 on linux. <details><summary>:mag_right: 6 errored, 5 changed</summary>

 * Error in 1b0ee2e07ac6: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 0828ac733081: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 1b0ee2e07ac6: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 0828ac733081: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 1b0ee2e07ac6: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Error in 0828ac733081: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Changed: **Tab_Bar-Custom_Buttons_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/Tab_Bar-Custom_Buttons_1-1b0ee2e07ac6.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/Tab_Bar-Custom_Buttons_1-0828ac733081.png)  
   Tab_Bar-Custom_Buttons_1 comparison
 * Changed: **Tab_Bar-Custom_Buttons_0** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/Tab_Bar-Custom_Buttons_0-1b0ee2e07ac6.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/Tab_Bar-Custom_Buttons_0-0828ac733081.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/Tab_Bar-Custom_Buttons_0.png)
 * Changed: **Icon-Icon_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/Icon-Icon_Image-1b0ee2e07ac6.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/Icon-Icon_Image-0828ac733081.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/Icon-Icon_Image.png)
 * Changed: **App_Tab_Bar-Search_Active** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/App_Tab_Bar-Search_Active-1b0ee2e07ac6.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/App_Tab_Bar-Search_Active-0828ac733081.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/App_Tab_Bar-Search_Active.png)
 * Changed: **App_Tab_Bar-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/App_Tab_Bar-Normal-1b0ee2e07ac6.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/App_Tab_Bar-Normal-0828ac733081.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497645152245-1b0ee2e07ac6-0828ac733081-linux/App_Tab_Bar-Normal.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits bef41d64bf1e...972c87dddbf6 introduce visual changes on linux. <details><summary>:mag_right: 5 changed</summary>

 * Changed: **Tab_Bar-Custom_Buttons_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/Tab_Bar-Custom_Buttons_1-bef41d64bf1e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/Tab_Bar-Custom_Buttons_1-972c87dddbf6.png)  
   Tab_Bar-Custom_Buttons_1 comparison
 * Changed: **Tab_Bar-Custom_Buttons_0** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/Tab_Bar-Custom_Buttons_0-bef41d64bf1e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/Tab_Bar-Custom_Buttons_0-972c87dddbf6.png)  
   Tab_Bar-Custom_Buttons_0 comparison
 * Changed: **Icon-Icon_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/Icon-Icon_Image-bef41d64bf1e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/Icon-Icon_Image-972c87dddbf6.png)  
   Icon-Icon_Image comparison
 * Changed: **App_Tab_Bar-Search_Active** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/App_Tab_Bar-Search_Active-bef41d64bf1e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/App_Tab_Bar-Search_Active-972c87dddbf6.png)  
   App_Tab_Bar-Search_Active comparison
 * Changed: **App_Tab_Bar-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/App_Tab_Bar-Normal-bef41d64bf1e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998140850-bef41d64bf1e-972c87dddbf6-linux/App_Tab_Bar-Normal-972c87dddbf6.png)  
   App_Tab_Bar-Normal comparison
</summary>

**Comment by chromakode** [comment]
Looking at visdiff (e.g. App_Tab_Bar-Search_Active), it looks like the badges lost their outline w/ background color:

image

image

**Comment by chrisnojima** [comment]
Yeah that's fixed in the mobile one. Didn't back port it

File: shared/common-adapters/tab-bar.desktop.js [hunk 1]
```diff
@@ -130,25 +136,36 @@ class TabBarButton extends Component<void, TabBarButtonProps, void> {
...
+              color: this.props.selected ? globalColors.white : globalColors.blue3_40,
+              ...this.props.styleIcon,
+            }}
+          />
           {badgeNumber > 0 &&
             <Box style={{...styleBadgeNav}}>
               <Badge
                 badgeNumber={badgeNumber}
                 badgeStyle={{marginLeft: 0, marginRight: globalMargins.tiny}}
-                outlineColor={globalColors.midnightBlue}
```
Comment (chromakode) [inline shared/common-adapters/tab-bar.desktop.js]: :point_up: Probably want to add this back w/ the new background color.

**Comment by chrisnojima** [comment]
I think this is in the next pr

**Pull request closed without merging**